### PR TITLE
Setup local dev

### LINKED ISSUE #17: Setup buy.u.c for local dev and add to source control

### PR DESCRIPTION
**Pull request by sowasred2012**
## Done

- Added current buy.ubuntu.com theme to the repo
- Set up the project for local development
- Updated readme with instructions for local development

## QA

- Clone this repo, and checkout this branch
- Follow the instructions in the readme
- See that a browser tab opens automatically and presents a version of the Shopify theme running on [localhost:3000](http://localhost:3000/)

Fixes https://github.com/canonical-web-and-design/ubuntu.com/issues/5781

### CODE REVIEW

**Comment by Lukewh** [comment]
:+1: The only thing I'd note is that it starts as https://localhost:3000 and requires you to accept the self signed certificate... Not sure if there's any way to serve as http?

**Review by Lukewh — APPROVED** [review]
Imma :+1: the code, but I've not read the 18,366 lines.

File: src/assets/ubuntu_white_hex.svg [hunk 1]
```diff
@@ -0,0 +1,2 @@
+<svg version="1.1" id="ubuntu-logo" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 757.3 170.1">
```
Comment (nottrobin) [inline src/assets/ubuntu_white_hex.svg:1]: Can assets not be linked from the assets server in this context?

Comment (sowasred2012) [reply src/assets/ubuntu_white_hex.svg:1]: They can, just at the time the asset manager search wasn't working, so I grabbed the inline SVG from u.c nav for the sake of getting this PR in. Will update this as part of the next PR.

Comment (nottrobin) [reply src/assets/ubuntu_white_hex.svg:1]: Thanks

File: README.md [hunk 2]
```diff
@@ -1,2 +1,21 @@
...
+- Clone this repo
+- Visit https://canonical-staging.myshopify.com/admin
+  - Click "Apps" in the left-hand nav
+  - At the bottom of the page, click "Manage private apps"
+  - From the list that appears, click "buy-ubuntu-theme-dev"
+  - Reveal and copy the contents of the password field
+- In the root of the repo, create a `config.yml` with the following content:
+
+```yml
+development:
```
Comment (nottrobin) [inline README.md:15]: It's a more common pattern for us to keep secrets and environment settings as environment variables, and then have the app read those environment variables. Is this at all possible in this project?

**Pull request merged**